.github/workflows/run_tests.yml

```diff
@@ -1,39 +1,46 @@
 name: Run tests
```

Comment:
Please revert these file changes and only add `if: "!contains(github.event.head_commit.message, 'skipci')"` and `actions/checkout@v2` and `shivammathur/setup-php@v2`. Just like the static analysis one. We don't use matrix testing in this package.